tests/Unit/ControlSystem/Actions/Test_InitializeMeasurements.cpp

```diff
@@ -234,11 +257,25 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.InitializeMeasurements",
   CAPTURE(db::get<Tags::Time>(box));
   CHECK(events_and_dense_triggers.is_ready(make_not_null(&box), cache, 0,
                                            component_p) ==
-        evolution::EventsAndDenseTriggers::TriggeringState::
-            NeedsEvolvedVariables);
+        (ab_active or c_active
+             ? evolution::EventsAndDenseTriggers::TriggeringState::
+                   NeedsEvolvedVariables
+             : evolution::EventsAndDenseTriggers::TriggeringState::Ready));
+
+  Submeasurement<tmpl::list<SystemA, SystemB>>::call_count = 0;
+  Submeasurement<tmpl::list<SystemC>>::call_count = 0;
   events_and_dense_triggers.run_events(box, cache, 0, component_p);
 
-  CHECK(Submeasurement<tmpl::list<SystemA, SystemB>>::call_count == 1);
-  CHECK(Submeasurement<tmpl::list<SystemC>>::call_count == 1);
+  CHECK(Submeasurement<tmpl::list<SystemA, SystemB>>::call_count ==
+        (ab_active ? 1 : 0));
+  CHECK(Submeasurement<tmpl::list<SystemC>>::call_count == (c_active ? 1 : 0));
+}
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.InitializeMeasurements",
+                  "[ControlSystem][Unit]") {
+  test_initialize_measurements(true, true);
```

Comment:
[optional] I know it's the same number of lines, but it's more easy to generalize in the future and it's clear what the two arguments are for.
```cpp
  for (const auto& [ab_active, c_active] :
       cartesian_product(make_array(true, false), make_array(true, false))) {
    test_initialize_measurements(ab_active, c_active);
  }
```

Reply:
No real reason. Just copied this from elsewhere that used the `cartesian_product`